src/FixedDataTable.js

```diff
@@ -849,6 +855,7 @@ class FixedDataTable extends React.Component {
         )}
         role="grid"
         aria-rowcount={ariaRowCount}
+        aria-multiselectable={isMultiselectable}
```

Comment:
I am wondering if this should also be a getter callback prop so that you can apply any dom attributes that you want. Since the FDT as no concept of selection.

Reply:
So should I just replace `isMultiselectable` with a callback method called `gridAttributesGetter` that does basically the same attributes assignments, but for the grid?

Reply:
sounds good